recipes.mdwn

```diff
@@ -33,6 +33,7 @@ to improve your Awesome setup.
 
 ## Themes
 
+* [Aesthetic Night](https://github.com/rxyhn/dotfiles)
```

Comment:
not neccessary but would be just nice if you'd add couple of words here about your theme, its features

Reply:
@awesomeWM/write-access should be consider posting one small-sized screenshot thumbnail near each receipt btw? i remember old version of that page in the wiki table had those

Reply:
alright sir, working on it

Reply:
> @awesomeWM/write-access should be consider posting one small-sized screenshot thumbnail near each receipt btw? i remember old version of that page in the wiki table had those

page not found, so i should make a screenshot?

Reply:
ignore the second question, it's not for you :)